tgui/packages/tgui/stories/Themes.stories.jsx

```diff
@@ -13,7 +13,7 @@ export const meta = {
 };
 
 const Story = (props) => {
-  const [theme, setTheme] = useLocalState('kitchenSinkTheme');
```

Comment:
and this one as well

Reply:
Applied with https://github.com/tgstation/tgstation/pull/80459/commits/6ec5fc718fe81976fc95fc9468452aa249172ffe